### PR TITLE
Rewording and updating README.md

### PR DESCRIPTION
**Pull request by ewolinetz**
@soltysh @bparees PTAL

### CODE REVIEW

**Comment by bparees** [comment]
lgtm.
[merge]

File: README.md [hunk 1]
```diff
@@ -19,17 +19,17 @@ Want to just get started now? Check out the [instructions](#getting-started).
 
 1. Simplify the process of application source + builder image -> usable image for most use cases (the
    80%)
-1. Define and implement a workflow for incremental build that eventually uses only docker
+1. Define and implement a workflow for incremental builds that eventually uses only Docker
```
Comment (soltysh) [inline README.md]: builds that eventually use~~s~~ only

File: README.md [hunk 2]
```diff
@@ -71,15 +71,15 @@ For detailed description of the requirements and the scripts along with examples
...
+Docker image with ONBUILD instructions and choosing the OnBuild strategy. This
 strategy will trigger all ONBUILD instructions and execute the assemble script
 (if it exists) as the last instruction.
 
 Since the ONBUILD images usually don't provide any entrypoint, in order to use
-this build strategy you have to provide it. You can either include the 'run',
+this build strategy you will have to provide one. You can either include the 'run',
 'start' or 'execute' script in your application source root folder or you can
-specify a valid STI scripts URL and the 'run' script will be fetched and set as
+specify a valid STI script URL and the 'run' script will be fetched and set as
```
Comment (soltysh) [inline README.md]: Actually the scripts should be in plural here, we are always referring to multiple scripts (at least 2 are required: `run` and `assemble` but additional 2 are possible).

File: README.md (hunk 1 above)
Comment (bparees) [inline README.md]: uses is correct... the noun "uses" applies to is "workflow" and not "builds", i believe.  or at least that argument can be made..it's a confusing sentence :)

File: README.md (hunk 2 above)
Comment (bparees) [inline README.md]: eh, again debatable.  it's a single script url.  what is located at that url can be one or more scripts.

just like a "source url" contains sources but we don't call it a sources url.

Comment (soltysh) [inline README.md]: ok

File: README.md (hunk 1 above)
Comment (soltysh) [inline README.md]: Right, after looking once again I see how that applies to workflow in which case it's correct. Sorry for the confusion.

**Pull request merged**